documentation/modules/proc-deploying-address-space-controller.adoc

```diff
@@ -9,11 +9,11 @@ The Address Space Controller is responsible for creating the infrastructure used
 
 .Prerequisites
 
-* Modify the installation files according to the namespace the Address Space Controller is going to be installed in
+* (Optional) Replace namespace if you want to deploy to something other than `{ProductNamespace}` (and substitute `{ProductNamespace}` in subsequent steps):
```

Comment:
I think this (Optional) step should be step one in the procedure section (with the same wording as the download section above).